hmda-analytics/src/main/scala/hmda/analytics/HmdaAnalyticsApp.scala

```diff
@@ -110,9 +111,24 @@ object HmdaAnalyticsApp
           for {
 
             delete <- transmittalSheetRepository.deleteByLei(ts.lei)
-            insert <- transmittalSheetRepository.insert(ts)
 
-          } yield insert
+          } yield delete
+        }
+        .runWith(Sink.ignore)
+    def step1: Future[Done] =
+      readRawData(submissionId)
```

Comment:
Do we never store the parsed ts data in cassandra? We certainly want to minimize what we write to cassandra but it feels strange that we are duplicating the parsing step when we write to postgres.

Reply:
parsed ts data is stored in cassandra. these steps aren't parsing the data but rather inserting data into PG. The raw data from cassandra needs to be parsed into TransmittalSheet prior to inserting into PG

Reply:
This may just be a misunderstanding of how the data is stored on my part but we do call `TsCsvParser` on line 122. If we are also storing the parsed TS couldn't we pull that from cassandra, instead of the data in csv form, and skip that step?

Reply:
#2505